diff --git a/bin/build-administration.sh b/bin/build-administration.sh
index 661de89..ac4ae34 100755
--- a/bin/build-administration.sh
+++ b/bin/build-administration.sh
@@ -43,6 +43,6 @@ else
     echo "Cannot check extensions for required npm installations as jq is not installed"
 fi
 
-(cd "${ADMIN_ROOT}"/Resources/app/administration && npm clean-install && npm run build)
+(cd "${ADMIN_ROOT}"/Resources/app/administration && npm install --force && npm run build)
 [[ ${SHOPWARE_SKIP_ASSET_COPY-""} ]] ||"${BIN_TOOL}" asset:install
 
